guvnor-asset-mgmt/guvnor-asset-mgmt-client/src/main/java/org/guvnor/asset/management/client/editors/project/structure/widgets/ProjectModulesViewImpl.java

```diff
@@ -94,31 +95,29 @@ public void setMode( ViewMode mode ) {
     }
 
     @Override
-    public void enableActions( boolean value ) {
+    public void enableActions( final boolean value ) {
         addModuleButton.setEnabled( value );
         actionsEnabled = value;
     }
 
     private void addModuleColumn() {
-
-        //Column<ProjectModuleRow, ?> column
```

Comment:
Good. 
If the code is commented out it is not used. If it is left there to be used later, then why make a PR if the feature is not ready.
Best to just delete commented code when we run into it.